client/examples/batchReceive.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 import { EventHubClient, EventData, EventPosition } from "../lib";
-import * as dotenv from "dotenv";
+import dotenv from "dotenv";
```

Comment:
Most of the samples were not running. So, @ShivangiReja has been working on fixing them. Those changes are in now and therefore conflicting with your changes here.

Can you please get the latest from the master branch and resolve the merge conflicts?

Reply:
Yes, I am aware of that PR and ready for rebasing onto it.

Reply:
Fixed.